implementation/src/test/java/io/smallrye/mutiny/groups/MultiDisjointTest.java

```diff
@@ -163,7 +163,6 @@ public void testFlatMapRequestsWithEmissionOnExecutor() {
                 .request(1);
 
         subscriber.awaitNextItem()
-                .request(2)
```

Comment:
what is the reason behind this removal?

Reply:
`awaitNextItems` makes a request, so here you'd request 4 elements, not 2.

Reply:
oh... ok...
I wrote that stuff and totally forgot.